pyproject.toml

```diff
@@ -174,7 +174,7 @@ build-backend = "poetry.core.masonry.api"
 
     [tool.poe.tasks.insert-config]
     help = "Inserts the configs into the database"
-    cmd = "python scripts/insert_simple_simulation.py"
+    cmd = "python scripts/insert_slightly_complicated_simulation.py"
```

Comment:
Should this change?

Reply:
Does not matter, because both should work. But maybe for consitency I will revert this.